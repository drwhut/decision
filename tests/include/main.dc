[Include("library.dc")]

Start~#1

< Set the library's variables >
Set(libInteger, #1, 9000)~#2
Set(libFloat, #2, 0.5)~#3
Set(libString, #3, "Wow.")~#4
Set(libBoolean, #4, true)~#5

< Print the library's variables >
libInteger~#6
Print(#5, #6)~#7

libFloat~#8
Print(#7, #8)~#9

libString~#10
Print(#9, #10)~#11

libBoolean~#12
Print(#11, #12)~#13

< Use the library's function >
IsEven(56783956)~#14
Print(#13, #14)~#15

< Use the library's subroutine >
SendLetter(#15, "Happy Birthday!", "drwhut")~#16