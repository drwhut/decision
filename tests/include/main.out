9000
0.5
Wow.
true
true
Happy Birthday!

From,
drwhut
